content/ja/docs/reference/glossary/kube-apiserver.md

```diff
@@ -19,4 +19,4 @@ tags:
 
 Kubernetes APIサーバーの主な実装は[kube-apiserver](/docs/reference/generated/kube-apiserver/)です。
 kube-apiserverは水平方向にスケールするように設計されています&mdash;つまり、インスタンスを追加することでスケールが可能です。
-複数のkube-apiserverインスタンスを実行することで、インスタンス間でトラフィックを分散させることが可能です。
\ No newline at end of file
+複数のkube-apiserverインスタンスを実行することで、インスタンス間でトラフィックを分散させることが可能です。
```

Comment:
not a big deal but can you tell what is the diff here?

Reply:
Sorry, I don't know why the diff is here...
I think I didn't touch this line.

Reply:
Please undone the diff if possible. If it is necessary that is okay but let me know.

Reply:
I have been able to remove this diff.